book.rb

```diff
@@ -0,0 +1,18 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] The `Rental` class is the intermediate class between the `Book` class and the `Person` Class. This implies that the `Book` and `Person` classes have a **_many-to-many_** relationship through the `Rental` class. **A book can be rented by many people and a person can rent many books**. To implement this, you need to create a new `Rental` class whenever you want to a `Book` or `Person` object wants to perform a rental. Kindly implement this association correctly. 👍 💻 

#### Hint

```ruby
def add_rental(person, date)
  Rental.new(date, self, person)
end
```